colour/models/rgb/rgb_colourspace.py

```diff
@@ -345,6 +490,100 @@ def decoding_cctf(self, value):
                     'decoding_cctf', value))
         self._decoding_cctf = value
 
+    @property
+    def use_derived_RGB_to_XYZ_matrix(self):
+        """
+        Property for **self._use_derived_RGB_to_XYZ_matrix** private attribute.
+
+        Returns
+        -------
+        bool
+            self._use_derived_RGB_to_XYZ_matrix.
+        """
+
+        return self._use_derived_RGB_to_XYZ_matrix
+
+    @use_derived_RGB_to_XYZ_matrix.setter
+    def use_derived_RGB_to_XYZ_matrix(self, value):
+        """
+        Setter for **self._use_derived_RGB_to_XYZ_matrix** private attribute.
+
+        Parameters
+        ----------
+        value : bool
+            Attribute value.
+        """
+
+        assert isinstance(value, bool), (
```

Comment:
It is probably part of a bigger discussion on what we allow users to do in general in our properties, so far the behaviour in the API has been to allow for less than more.

In that context, I'm a bit uncomfortable at the idea that one could stuff literally any object in the underlying attribute.